api-report/container-runtime.api.md

```diff
@@ -123,7 +123,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     getAudience(): IAudience;
     getCurrentReferenceTimestampMs(): number | undefined;
     getGCData(fullGC?: boolean): Promise<IGarbageCollectionData>;
-    getGCNodePackagePath(nodePath: string): readonly string[] | undefined;
+    getGCNodePackagePath(nodePath: string): Promise<readonly string[] | undefined>;
```

Comment:
Is this not a breaking change?

Reply:
Not really. It's not on a public interface.